config/crds/hive.openshift.io_hiveconfigs.yaml

```diff
@@ -114,6 +114,11 @@ spec:
                               QPS for a controller
                             format: int32
                             type: integer
+                          replicas:
+                            description: Replicas specifies the number of replicas
+                              the specific controller pod should use.
```

Comment:
Could you elaborate here that this is only for controllers which have been split out to run in their own pods. For others it will be ignored.